src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -441,7 +441,9 @@ public class Options
             public bool AllowQuoting = Defaults.AllowQuoting;
 
             /// <summary>
-            /// Whether the input may include sparse representations.
+            /// Whether the input may include sparse representations. For example, a row containing
+            /// "5 2:6 4:3" means that there are 5 columns, and the only non-zero are columns 2 and 4, which have values 6 and 3,
```

Comment:
>5 2:6 4:3 [](start = 17, length = 9)

The "5" can also appear in the middle of the line. So, for example, the file can contain this line:
1 2 10 2:3 7:4
this maps to a length 12 vector with the values 1, 2, 0, 0, 3, 0, 0, 0, 0, 4, 0, 0.
So the "10" in that line refers to the number of sparse elements **after** the dense values, indexed accordingly (in this case, index 0 would be the third element in the vector, index 1 the fourth and so on).

You should also mention the connection to the argument `InputSize`, which should be used when the file doesn't contain the number of sparse elements in every line like in "5 2:6 4:3" or in my example above. Note though, that when using `InputSize`, the specified size should be the **total** size, not just the size of the sparse part, but the indices are relative to where the sparse part begins. For example, if `InputSize=10`, then the line 
1 2 2:3 7:4 
would be mapped to 1,2,0,0,3,0,0,0,0,4. And if `InputSize=12`, then the same line would be mapped to 1,2,0,0,3,0,0,0,0,4,0,0.